HunterPie.Core/Game/Data/MonsterData.cs

```diff
@@ -69,6 +69,7 @@ private static void ParseAllMonsters()
                 MonsterDataSchema schema = new()
                 {
                     Id = int.Parse(monster.Attributes["Id"].Value),
+                    Capture = int.Parse(monster.Attributes["Capture"].Value),
```

Comment:
HunterPie uses the same Monster Data parser for both MHW and MHR, so having the field "Capture" as required will crash HunterPie since MHR data doesn't have that.
```suggestion
                    Capture = int.Parse(monster.Attributes["Capture"]?.Value ?? "0"),
```

Reply:
Good catch (I don't have MHR yet)! This is now fixed.